test/integration/Region.e2e.test.js

```diff
@@ -11,31 +11,37 @@ describe('Regions', () => {
         // Wait for the empty region layer to be present
         cy.getByTestId('ba-Layer--region');
 
-        // Assert that the region creator does not exist and no annotations are present
+        // Assert that the region creator does not exist
         cy.getByTestId('ba-RegionCreator').should('not.exist');
-        cy.get('[data-testid^="ba-AnnotationTarget"]').should('not.exist');
 
-        // Enter region creation mode
-        cy.getByTestId('bp-AnnotationsControls-regionBtn').click();
+        cy.document().then(document => {
+            // Record the number of annotations
+            const { length } = document.querySelectorAll('[data-testid^="ba-AnnotationTarget"]');
 
-        // Add a region annotation on the document
-        cy.drawRegion();
-        cy.submitReply();
+            // Enter region creation mode
+            cy.getByTestId('bp-AnnotationsControls-regionBtn').click();
 
-        // Assert that at least one annotation is present on the document and is active
-        cy.get('[data-testid^="ba-AnnotationTarget"]').should('have.class', 'is-active');
+            // Add a region annotation on the document
+            cy.drawRegion();
+            cy.submitReply();
+
+            // Assert that one more annotation is created on the document
+            cy.get('[data-testid^="ba-AnnotationTarget"]').should('have.length', length + 1);
+        });
```

Comment:
Is this change (find the length, and then make sure length + 1 is the result) needed because there are Regions and Highlights being executed during the test run now?

Reply:
yeah